### ISSUE TITLE
Security - Create 'phar' so this can be run independently

### ISSUE THREAD

**Issue by fredden** [sansecio/composer-integrity-plugin#12]
As this is intended as a security analysis tool, installing this into a production environment in order to run it against the code-base doesn't necessarily make sense. If an attacker has the ability to modify files in `vendor/` (such that they would then fail the validation check), they also have the ability to modify the output from this tool.

In order to allow an auditor to reliably run this tool, please can you package this as a 'phar' command so that a single file can be (verified and then) uploaded to the production server to perform the analysis.

(It would be helpful to have a 'phar-only' Composer package for ease, but that's not a requirement here. See https://github.com/netz98/n98-magerun2-dist as an example.)

Also, the installation instructions suggest installing this as a development dependency, so the tool won't be available on production environments. It seems less helpful to run this tool on a development environment, where the opportunity for an attacker to modify files is lower. I think the intention is to run this tool on a production environment.

**Comment by danslo** [comment]
Having a phar makes sense.

I've made the necessary preparations to be able to use this code without being part of a composer command in this PR: https://github.com/sansecio/composer-integrity-plugin/pull/14

There's still some additional work that needs to be done to create a phar (probably using [box](https://github.com/box-project/box)).

This will probably take a couple of days, bear with me :)

**Comment by danslo** [comment]
Just released version [0.2.0](https://github.com/sansecio/composer-integrity-plugin/releases/tag/0.2.0), which is built as a phar.

The only difference is that it doesn't detect patches (because we must hook into composer internals to figure this out).
If someone ever makes that feature request, we can maybe consider it :)

**Comment by Qjammer** [comment]
The phar does not seem to be working properly. When run on an empty directory, it produces the following error: 

```
Fatal error: Uncaught DI\NotFoundException: No entry or class found for 'Symfony\Component\Console\Application' in phar:///application/mytest/composer-integrity.phar/vendor/php-di/php-di/src/Container.php:133
Stack trace:
#0 phar:///application/mytest/composer-integrity.phar/composer-integrity.php(12): DI\Container->get('Symfony\\Compone...')
#1 /application/mytest/composer-integrity.phar(14): require('phar:///applica...')
#2 {main}
  thrown in phar:///application/mytest/composer-integrity.phar/vendor/php-di/php-di/src/Container.php on line 133

```

It appears it does not have the `symfony/console` package bundled, so it cannot run.

If I run it in a directory with a pre-existing `./vendor` directory, then it produces the following error:

```
Fatal error: Uncaught Error: Class "DI\Container" not found in phar:///application/composer-integrity.phar/composer-integrity.php:10
Stack trace:
#0 /application/composer-integrity.phar(14): require()
#1 {main}
  thrown in phar:///application/composer-integrity.phar/composer-integrity.php on line 10
```

I messed around with it and figured out that the phar is `require_once`  the outer `vendor/autoload.php`, instead of the bundled one, thus it can't find the class `DI\Container`, since I don't have it installed.
